scrapers/reddit.js

```diff
@@ -0,0 +1,56 @@
+var BaseLoklakScrapper = require('./base');
+
+class RedditLoklakScrapper extends BaseLoklakScrapper {
+
+    constructor() {
+        super('Reddit', 'https://www.reddit.com');
+    }
+
+    argumentSanityCheck(args) {
+        super.argumentSanityCheck(args);
+
+        if (args.length <= 2) {
+            console.error('Atleast one argument required.');
+            process.exit(-1);
+        }
+
+        return true;
+    }
+
+    onInit() {
+        var profile = process.argv[2];
+        this.REQUEST_URL = this.BASE_URL + "/r/" + profile;
+        this.request();
+    }
+
+    scrape($) {
+        var redditEntries = [];
+        var entries = $(".thing");
+        var BASE_URL = this.BASE_URL;
+
+        entries.each(function (i, elem) {
```

Comment:
@hemantjadon thanks for pointing :) arrow function will solve the 'this' problem.